bukkit/build.gradle

```diff
@@ -5,6 +5,7 @@ dependencies {
     implementation 'io.papermc:paperlib:1.0.7'
     implementation 'me.lucko:commodore:2.2'
     implementation 'net.kyori:adventure-platform-bukkit:4.1.2'
+    implementation 'com.github.Emibergo02:RedisEconomy:master-SNAPSHOT'
```

Comment:
I'd rather target a stable version for build reproducible if possible :)

Reply:
edited master to 1.0-SNAPSHOT